README.md

```diff
@@ -34,6 +34,7 @@ plugins: [
           tableName: `YOUR_TABLE_NAME`,
           tableView: `YOUR_TABLE_VIEW_NAME`, // optional
           queryName: `OPTIONAL_NAME_TO_IDENTIFY_TABLE`, // optional
+          createSeparateNodeType: true, // optional - makes all records in this table a separate node type, based on your tableView, or if not present, tableName, e.g. a table called "Fruit" would become "allAirtableFruit"
```

Comment:
Might be worth mentioning that the default is false. Also, we should probably add another section in the docs on _why_ you would care to do this. We could even point back to the issues regarding it.